sklearn/linear_model/_logistic.py

```diff
@@ -973,6 +973,7 @@ def _log_reg_scoring_path(X, y, train, test, pos_class=None, Cs=10,
         max_squared_sum=max_squared_sum, sample_weight=sample_weight)
 
     log_reg = LogisticRegression(solver=solver, multi_class=multi_class)
+    log_reg.n_features_in_ = X.shape[1]
```

Comment:
Related to https://github.com/scikit-learn/scikit-learn/pull/18011

Reply:
and https://github.com/scikit-learn/scikit-learn/pull/18577#pullrequestreview-505721707